change/@fluentui-react-message-bar-preview-a6a0acb0-3fd7-4b96-b883-fbd5711cc593.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29462&commitId=5d554207109bab61cfba1acf0f0e35e546cbc173&env=prod&clientType=FLUENTUI)


  <details><summary><b>MessageBar</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MessageBar.Auto.default.chromium.png|551| Changed|
|MessageBar.Multiline.default.chromium.png|1104| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
It would be great if you could crop the stories to avoid the white space.

Reply:
Yeah will do that in a follow up PR